src/WebJobs.Script.WebHost/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -49,7 +49,6 @@ public static async Task<FunctionMetadataResponse> ToFunctionMetadataResponse(th
             {
                 var testDataFilePath = functionMetadata.GetTestDataFilePath(hostOptions);
                 response.TestDataHref = VirtualFileSystem.FilePathToVfsUri(testDataFilePath, baseUrl, hostOptions);
-                response.TestData = await GetTestData(testDataFilePath, hostOptions);
```

Comment:
Delete property from: azure-functions-host\src\WebJobs.Script.WebHost\Models\FunctionMetadataResponse.cs?

Reply:
It's still used on PUTs